### DISCUSSION TITLE
Should we start new ecosystem program in DCDAO to support two roles: Integration Specialist and Data Analyst?

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#55] [Ideas]
Launching a new program in DCDAO to fund two specialist roles, an Integration Specialist and a Data Analyst, is a strategic move to enhance the ecosystem's capabilities. 

Since these roles were previously supported by Aki from Itering, who is now focusing on Developer Relations and Education, filling these vacancies is crucial for maintaining and improving the ecosystem's functions. 

Here's a suggested approach to launching this program and attracting suitable candidates:

### Responsibilities:

- Oversee and manage Darwinia integration projects on a part-time basis, coordinating closely with external systems.
- Collaborate with technical teams to address and solve integration challenges, contributing to the ecosystem's development in a flexible capacity.
- The current projects demanding integration involve a range of critical services and providers in the ecosystem, including Indexers such as TheGraph, Subquery, and Subsquid; Developer tools; RPC (Remote Procedure Call) providers; various Data Providers like Coingecko; and Wallet Providers, among others..

### Qualifications:

- Proven experience in system integration, ideally in a part-time or project-based context.
- Technical proficiency in relevant programming languages and tools, adaptable to part-time work schedules.

### Responsibilities:

- Collaboration with Data Platforms:

  - Work part-time with Subscan Open Platform and other key data platforms, focusing on data collection, interpretation, and analysis.
  - Ensure effective data integration and exchange between DCDAO and these platforms in a flexible working arrangement.

- Community-Driven Initiatives:

  - Employ data analysis skills to understand and support community needs and engagement, tailored to a part-time role.
  - Provide analytical support to community projects and initiatives on a part-time basis.

### Qualifications:

- Strong analytical skills, ideally suited for part-time involvement.
- Proficiency in data analysis tools and methodologies, capable of delivering insights in a part-time capacity.

Given the current funding considerations and the nascent stage of the community talent pool, the roles of Integration Specialist and Data Analyst in the DCDAO ecosystem can be structured as part-time positions. This approach not only aligns with the resource availability but also offers flexibility, allowing for the engagement of talented individuals who may have other commitments. These roles will be overseen by the DCDAO Technical Manager, ensuring cohesive management and strategic alignment.

This program represents a major step forward in strengthening our ecosystem's infrastructure and community engagement. We invite talented individuals who are passionate about these areas to join us in this endeavor and contribute to the DCDAO's evolving journey.

**Comment by SasoLithops** [comment]
Hi!

If I understand correctly it is proposed for those two roles to be some kind of advisors or "helpers on demand". Will they be chosen among existing DCDAO members or someone who is yet to be one one but knows Darwinia well? Hope we get ones with great knowledge. 
Growth manager on the other hand is for permanent position right?

Will these new roles be funded from DCDAO treasury (I know at this time DCDAO is 100% dependent on Darwinia treasury, but in future) or will they be funded from Darwinia treasury? If they will be from Darwinia treasury we need to vote there too after approval on DCDAO snapshot.

I am glad DCDAO members will vote on their application so hope we learn more from them there, where they can also introduce themselves.

**Reply by hackfisher** [reply]
In the long term, DCDAO Treasury will probably be funded by Darwinia Treasury, as far as I know, @Cindy884 is drafting a proposal to apply funding from Darwinia Treasury for the budget of DCDAO 2024 Q1

**Reply by hackfisher** [reply]
@SasoLithops Are you interested to any of these positions here Integration Specialist or GrowthManager, they looks fit for you.

**Reply by SasoLithops** [reply]
GM!

Thank you. I am very honored for the offer, but sadly don't match qualifications nor posses the knowledge to apply in any of two positions. 
But will gladly help with some things I learned to more capable person.
I recomend Laki for Data analyst. He is really good at this has proven many times.
